template/cordova/lib/deployment.js

```diff
@@ -35,7 +35,7 @@ function run(cmd, args, opt_cwd) {
         child.stderr.on('data', function(s) { stderr += s; });
         child.on('exit', function(code) {
             if (code) {
-                d.reject(stderr);
+                d.reject({ message: stderr, code: code, toString: function() { return stderr; }});
```

Comment:
This isn't logging.  This is providing an object instead of a string to the failure case of a promise.